React/Modules/MacOS/RCTAccessibilityManager.m

```diff
@@ -39,11 +39,11 @@ - (instancetype)init
                                              selector:@selector(accessibilityDisplayOptionsChange:)
                                                  name:NSWorkspaceAccessibilityDisplayOptionsDidChangeNotification
                                                object:nil];
+    _isHighContrastEnabled = [[NSWorkspace sharedWorkspace] accessibilityDisplayShouldIncreaseContrast];
```

Comment:
Not your fault but while we're touching these can we cache this to save all the extra time calling into the same method over and over again?

`NSWorkspace *sharedWorkspace = [NSWorkspace sharedWorkspace` and just use that var throughout the method

Reply:
That would be a micro-optimization since calling these singleton methods take microseconds and I think it's fine to continue following the convention in React Native of calling them for each use (see `[NSNotificationCenter defaultCenter]` calls in other native modules).

Reply:
I'd argue those should get fixed too. Functionally speaking it will only improve performance (albeit a small amount) with no downsides if we fix up these calls everywhere.